plotter/scatterColor_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"testing"
+
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/internal/cmpimg"
+	"gonum.org/v1/plot/palette/moreland"
+	"gonum.org/v1/plot/vg"
+	"gonum.org/v1/plot/vg/draw"
+	"gonum.org/v1/plot/vg/vgimg"
+)
+
+// ExampleScatterColor draws some scatter points, a line,
+// and a line with points.
+func ExampleScatter_color() {
+	rnd := rand.New(rand.NewSource(1))
+
+	// randomPoints returns some random x, y points
+	// with some interesting kind of trend.
+	randomPoints := func(n int) XYs {
```

Comment:
Would it make sense for this function to return `XYZs` instead of `XYs`? Then we wouldn't need to separately create z values below.

Reply:
In this case we will have to include `Z` to the type `Scatter`, won't we?

Reply:
I think we can change it to be the same `randomTriples` function that is in the bubbles example.